desktop_version/src/Network.c

```diff
@@ -89,16 +107,22 @@ void NETWORK_shutdown(void)
     #endif
 }
 
-void NETWORK_update(void)
+
+int32_t NETWORK_update(void)
 {
+    int32_t result = 0;
     #if NUM_BACKENDS > 0
     int32_t i;
     for (i = 0; i < NUM_BACKENDS; i += 1)
     if (backends[i].IsInit)
     {
-        backends[i].Update();
+        if ( backends[i].Update() )
```

Comment:
We don't need to worry about Update status too much - as long as we do a good job with SetRPC this can stay as-is.